bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -378,7 +380,7 @@ spec:
                 - --enable-leader-election
                 - --zap-log-level=info
                 - --zap-time-encoding=rfc3339nano
-                - --feature-gates=+operator.autoinstrumentation.go,+operator.autoinstrumentation.nginx
+                - --feature-gates=+operator.autoinstrumentation.go,+operator.observability.prometheus,+operator.autoinstrumentation.nginx
```

Comment:
Is this an intended change?

Reply:
No, I was debugging why the `podmonitors` were not being created and by accident, I pushed this change. I've just fixed and the e2e should be fine now.